.orchestra/ci/mass-testing-weekly/entrypoint.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -euo pipefail
+SCRIPT_DIR=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
+
+TIMESTAMP=$(date +%s)
+
+# Install dependencies and the orchestra tool
+sudo apt-get update
+sudo apt-get install -y less nano vim inkscape bc
+
+cd "$SCRIPT_DIR/../../.."
+.orchestra/ci/install-dependencies.sh
+pip3 -q install --user "git+https://github.com/revng/revng-orchestra.git@master"
+export PATH="$HOME/.local/bin:$PATH"
+
+# Copy user_options.yml and update
+cp -a "$SCRIPT_DIR/user_options.yml" .orchestra/config/user_options.yml
+orc update --no-config
+
+# Run the progress poller, which will print the progress of mass-testing every
+# hour
+"$SCRIPT_DIR/progress_poller.sh" &
+POLLER_PID=$!
+
+# Run the actual mass-testing
+RC=0
+orc --quiet install mass-testing || RC=$?
+
+kill -9 "$POLLER_PID"
+
+# If mass-testing failed in any way, sleep forever. This is to allow manual
+# intervention to recover the data (if present), since each run takes multiple
+# days to finish
+if [ "$RC" -ne 0 ]; then
+  while true; do
+    sleep 24h
+  done
+  exit "$RC"
+fi
+
+# Configure ssh for rsync-ing
```

Comment:
Clearly state what we're doing here.

We're publishing the mass.rev.ng website, right?